test/units/testsuite-74.type-exec-parallel.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Maybe run this as part of TEST-07-PID1 instead (i.e. rename the file to `testsuite-07.type-exec-parallel.sh`, given it checks PID1 behavior?